getting-started/gopher-run/generator/generator.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package generator returns procedurally generated level for Gopher Run
+package generator
```

Comment:
Please add tests for this package. One test for the top-level exported function is probably enough. Test template:

```
package generator

// import ...

func TestGenerateObject(t *testing.T) {
	// Do something with GenerateObject.
	if ... {
		t.Errorf("GenerateObject got %v, want %v", ...)
	}
}
```

See https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures for some helpful info.